src/workflows/dags/meta_data.py

```
@@ -45,7 +43,7 @@ def extract_sites() -> pd.DataFrame:
 
     @task()
     def load_sites(data: pd.DataFrame):
-        from airqo_etl_utils.bigquery_api import BigQueryApi
+        from airqo_etl_utils.bigquery_api import BigQueryAp
```

Ruff (0.8.2)

`airqo_etl_utils.bigquery_api.BigQueryAp` imported but unused

Remove unused import: `airqo_etl_utils.bigquery_api.BigQueryAp`

(F401)